code/modules/asset_cache/asset_list_items.dm

```diff
@@ -136,6 +136,21 @@
 
 /datum/asset/spritesheet/mechaarmor/register()
 	InsertAll("", 'icons/UI_Icons/mecha/armor.dmi')
+	..()
```

Comment:
I think we enforce . = ..(), do you want to skip it on that ?

Reply:
Spritesheets are tg code and none of them do it so I kept it